src/models/bullet.js

```diff
@@ -276,6 +285,7 @@ nv.models.bullet = function() {
         width:    {get: function(){return width;}, set: function(_){width=_;}},
         height:    {get: function(){return height;}, set: function(_){height=_;}},
         tickFormat:    {get: function(){return tickFormat;}, set: function(_){tickFormat=_;}},
+        bulletDuration:    {get: function(){return bulletDuration;}, set: function(_){bulletDuration=_;}},
```

Comment:
This is just called duration on other charts, so it should match.  

See the "duration" option on most charts:   http://nvd3-community.github.io/nvd3/examples/documentation.html